documentation/config_file_auth_migration.markdown

```diff
@@ -0,0 +1,342 @@
+---
+layout: default
+title: "Puppet Server Configuration Files: Migrating to the HOCON auth.conf Format"
+canonical: "/puppetserver/latest/config_file_auth_migration.html"
+---
+
+[Puppet `auth.conf`]: /puppet/latest/reference/config_file_auth.markdown
+[deprecated]: ./deprecated_features.markdown
+[Backward Compatibility with Puppet 3 Agents]: ./compatibility_with_puppet_agent.markdown
+
+Puppet Server 2.2.0 introduced a significant change in how it manages authentication to API endpoints. The older [Puppet `auth.conf`][] file and whitelist-based authorization method are [deprecated][]. Puppet Server's new `auth.conf` file, illustrated below in examples, also uses a different format for authorization rules.
+
+Use the following examples and methods to convert your authorization rules when upgrading from Puppet Server 2.2.0 and newer. For detailed information about using the new or deprecated `auth.conf` rules with Puppet Server, see the [Puppet Server `auth.conf` documentation](./config_file_auth.markdown).
+
+> **Note:** To continue using the deprecated Ruby [Puppet `auth.conf`][] file and authorization methods, see the [Deprecated Ruby Parameters](/puppet/latest/reference/config_file_auth.markdown#deprecated-ruby-parameters) section of the `auth.conf` documentation. To support both Puppet 3 and Puppet 4 agents connecting to Puppet Server, see [Backward Compatibility with Puppet 3 Agents][].
+
+## Managing rules with Puppet modules
+
+You can reimplement and manage your authorization rules in the new HOCON format and `auth.conf` file by using the [`puppetlabs-puppet_authorization`](https://forge.puppet.com/puppetlabs/puppet_authorization) Puppet module. See the module's documentation for details.
+
+## Converting rules directly
+
+Most of the deprecated authorization rules and settings are available in the new format.
+
+> ### Unavailable rules, settings, or values
```

Comment:
It is (we use blockquote styles to break out asides and notes), but it's evolved into a big-enough thing to not need that formatting. I'll remove the blockquotes.